src/ansys/fluent/core/launcher/fluent_container.py

```diff
@@ -186,7 +186,7 @@ def configure_container_dict(
         if file_transfer_service:
             host_mount_path = pyfluent.USER_DATA_PATH
         else:
-            host_mount_path = pyfluent.EXAMPLES_PATH
+            host_mount_path = pyfluent.CONTAINER_MOUNT_PATH
```

Comment:
Should we, instead of using a module constant `pyfluent.CONTAINER_MOUNT_PATH`, use an environment variable called `PYFLUENT_CONTAINER_MOUNT_PATH`, as I initially suggested in https://github.com/ansys/pyfluent/pull/2874#discussion_r1622426411 ?

Something like this is what I had in mind:
```
host_mount_path = Path(os.getenv("PYFLUENT_CONTAINER_MOUNT_PATH", os.getcwd()))
```

Reply:
Done. Thank you.